CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+# v3.0.0 (2024-10-17)
+
+## OS Changes
+* Drop kubernetes-1.23 ([#184])
+* Set Epoch to 1 in necessary packages ([#180])
+* Move kmod-5.10-nvidia to R535 branch from R470 ([#181])
```

Comment:
```suggestion
* Move kmod-5.10-nvidia from branch R470 to R535 ([#181])
```

Reply:
I'm fine with the change. It reads more clearly.